packages/client/src/pages/CreateSafe.js

```diff
@@ -100,7 +101,7 @@ const AuthorizeTreasury = ({
 };
 
 function CreateSafe({ web3 }) {
-  const [safeType, setSafeType] = useState("Social");
+  const [safeType, setSafeType] = useState(SAFE_TYPES.SOCIAL);
```

Comment:
`SafeType` can be removed already. It's no longer in design. But it's ok to address it later when we implement new createSafe UI.